### PR TITLE
"Splash Commit"

### PR DESCRIPTION
**Pull request by DapedarKeerthana6**
SplashScreen Added

### CODE REVIEW

File: app/src/main/java/com/example/dazn_movie_player/SplashScreen.kt [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import android.os.Bundle
+import android.os.Handler
+import android.os.Looper
+
+class SplashScreen : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_splash_screen)
+
+        supportActionBar?.hide()
```
Comment (DayaDAZN) [inline app/src/main/java/com/example/dazn_movie_player/SplashScreen.kt]: You can take another theme for full screen view. Don't use action bar hide here.

File: app/src/main/java/com/example/dazn_movie_player/SplashScreen.kt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+        val handler = Handler(Looper.getMainLooper())
+        handler.postDelayed({
```
Comment (DayaDAZN) [inline app/src/main/java/com/example/dazn_movie_player/SplashScreen.kt]: Remove handler here and maintain in single line Handler(Looper.getMainLooper()).postDelayed({})

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -11,15 +11,25 @@
...
+            android:theme="@style/Theme.SplashScreen"
+            tools:ignore="WrongManifestParent">
+        <intent-filter>
+            <action android:name="android.intent.action.MAIN" />
+
+            <category android:name="android.intent.category.LAUNCHER" />
+        </intent-filter>
+        </activity>
+
         <activity
```
Comment (Ravali-Dazn) [inline app/src/main/AndroidManifest.xml:26]: Remove the activity code if not available

File: app/src/main/res/layout/activity_splash_screen.xml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/black"
+
```
Comment (DayaDAZN) [inline app/src/main/res/layout/activity_splash_screen.xml]: Remove extra space here

File: app/src/main/java/com/example/dazn_movie_player/SplashScreen.kt [hunk 5]
```diff
@@ -0,0 +1,25 @@
+package com.example.dazn_movie_player
```
Comment (Ravali-Dazn) [inline app/src/main/java/com/example/dazn_movie_player/SplashScreen.kt:1]: Better not to have underscores for package name

File: app/src/main/java/com/example/dazn_movie_player/SplashScreen.kt [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+import android.os.Looper
+
+class SplashScreen : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_splash_screen)
+
+        supportActionBar?.hide()
+
+        val handler = Handler(Looper.getMainLooper())
```
Comment (Ravali-Dazn) [inline app/src/main/java/com/example/dazn_movie_player/SplashScreen.kt]: Code format needed

File: app/src/main/res/layout/activity_splash_screen.xml [hunk 7]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (Ravali-Dazn) [inline app/src/main/res/layout/activity_splash_screen.xml:2]: Constraint layout is recommended, can use that going forward

File: app/src/main/AndroidManifest.xml (hunk 3 above)
Comment (Ravali-Dazn) [inline app/src/main/AndroidManifest.xml:24]: Can remove the activity code if not available

**Review by Ravali-Dazn — CHANGES_REQUESTED** [review]

File: app/src/main/AndroidManifest.xml (hunk 3 above)
Comment (Siva-Velankayala) [reply app/src/main/AndroidManifest.xml:24]: remove unused tags

File: app/src/main/java/com/example/dazn_movie_player/SplashScreen.kt (hunk 5 above)
Comment (Siva-Velankayala) [reply app/src/main/java/com/example/dazn_movie_player/SplashScreen.kt:1]: Use camel case notation

File: app/src/main/java/com/example/dazn_movie_player/SplashScreen.kt [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+        handler.postDelayed({
+            val intent = Intent(this,MainActivity::class.java)
+            startActivity(intent)
+            finish()
+        },3000)
```
Comment (Siva-Velankayala) [inline app/src/main/java/com/example/dazn_movie_player/SplashScreen.kt]: code format

File: app/src/main/res/layout/activity_splash_screen.xml [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/black"
+
+    tools:context=".SplashScreen">
+
+    <pl.droidsonroids.gif.GifImageButton
```
Comment (Siva-Velankayala) [inline app/src/main/res/layout/activity_splash_screen.xml:9]: Check if  you can achieve this without using any third party library

**Review by Siva-Velankayala — CHANGES_REQUESTED** [review]

**Pull request closed without merging**